_data/data.yml

```diff
@@ -14,6 +14,7 @@ sidebar:
     # Sidebar links
     email: hello@webjeda.com
     phone: 012 345 6789
+    timezone: America/Cancun Timezone
```

Comment:
I am not sure which spelling of timezone is preferred, there seems to be [open debate](https://english.stackexchange.com/questions/3934/time-zone-vs-timezone#:~:text=There%20seem%20to%20be%20three,%2Dzone%2C%20and%20time%20zone.&text=All%20three%20seem%20to%20be,case%20I%20would%20use%20timezone.) about what is the correct spelling. I am happy to make an update if anyone has a preference different than what is here.